ontology/cl.md

```diff
@@ -84,6 +84,12 @@ usages:
     title: 'The human body at cellular resolution: the NIH Human Biomolecular Atlas Program.'
   type: annotation
   user: https://hubmapconsortium.org/
+- description: The single-cell transcriptomics platform CZ CELLxGENE uses CL to annotate all cell types. All datasets on CellXGene are annotated according to a standard schema that specifies the use of CL to record Cell Type.
+  publications:
+  - id: https://doi.org/10.1101/2021.04.05.438318
+    title: 'CELLxGENE: a performant, scalable exploration platform for high dimensional sparse matrices'
+  type: annotation
+  user: https://cellxgene.cziscience.com/
```

Comment:
Is there an example URL you can provide that resolves to a webpage about a CL term? If so, it would be good to add that to the examples section.

Reply:
I can add two, one of a dataset where all clusters are nnotated to CL terms (which you can click for a popup window of cellguide that contains the CL ID hyperlinked) and the proper cellGuide page (contains  CL term that is being used by CELLxGENE, with the CL definition and their own ChatGPT generated description):
- https://cellxgene.cziscience.com/e/55003f67-c494-46f1-83fb-902745646379.cxg/
- https://cellxgene.cziscience.com/cellguide/CL:4033057

Reply:
the second is perfect, note that you can add it to an additional `examples` section the way you can see it for other usages

Reply:
Will do, thank you!

Reply:
I have added the example with a description, thank you for the suggestion!